dasharo-performance/core-performance-boost.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+
+Suite Setup         Prepare Test Suite
+Suite Teardown      Log Out And Close Connection
+
+
+*** Test Cases ***
+CPB001.001 Check if disabling CPB decrease performance
+    Skip If    not ${CPB_SUPPORT}    CBP not supported
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${apu_menu}=    Enter Dasharo Submenu    ${setup_menu}    Dasharo APU Configuration
+    Set Option State    ${apu_menu}    Core Performance Boost    ${FALSE}
+    Save Changes And Reset
+    Boot System Or From Connected Disk    debian
+    Login To Linux
+    ${first_check}=    Execute Linux Command
```

Comment:
Please try to use terminal keywords as suggested here: https://github.com/Dasharo/open-source-firmware-validation?tab=readme-ov-file#keywords
We have some redundancy in keywords and do not want to make it even more.

Reply:
Done.